Applications/Telemetry/telemetry.py

```diff
@@ -161,7 +199,6 @@ def dbTelemetry():
     parameters["TIMESPAN"] = timespan
     parameters["LIMIT"] = limit
 
-    data = []
```

Comment:
Assumes local variables are lost after function run-time and don't need re-initialization to clear old values.